default-configure ends with "configure: error: setcap not found"

**Describe the bug**
Trying to build current dosemu2 gives the error "configure: error: setcap not found" on executing `./default-configure` (regardless of whether I include my desired options or not).

I don't have root/sudo access on this machine.

**A regression?**
Not a problem for 6a2f4f527e4fb501196abec23351f221691086a2

**dosemu2 origins**
549cc5949d8cf7c270254e448bfa10671153995f from github

**Additional info**
Guessing that it may be related to the new "no priv sep" switch which I probably would want to use.

This just means libcap is there but
setcap not. You need also libcap-bin
or alike.

Суббота, 6 сентября 2025 г получено от E. C. Masloch:
> ecm-pushbx created an issue (dosemu2/dosemu2#2562)
> 
> **Describe the bug**
> Trying to build current dosemu2 gives the error "configure: error: setcap not found" on executing `./default-configure` (regardless of whether I include my desired options or not).
> 
> I don't have root/sudo access on this machine.
> 
> **A regression?**
> Not a problem for 6a2f4f527e4fb501196abec23351f221691086a2
> 
> **dosemu2 origins**
> 549cc5949d8cf7c270254e448bfa10671153995f from github
> 
> **Additional info**
> Guessing that it may be related to the new "no priv sep" switch which I probably would want to use.
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/dosemu2/dosemu2/issues/2562
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***

--- a/configure.ac
+++ b/configure.ac
@@ -354,7 +354,7 @@ PKG_CHECK_MODULES([LIBACL], [libacl], [
 )
 
 PKG_CHECK_MODULES([LIBCAP], [libcap], [
-    AC_PATH_PROGS([SETCAP], [setcap], [], [$PATH${PATH_SEPARATOR}/usr/sbin])
+    AC_PATH_PROGS([SETCAP], [setcap], [], [$PATH${PATH_SEPARATOR}/usr/sbin${PATH_SEPARATOR}/sbin])
     if test -z "$SETCAP"; then
       AC_MSG_ERROR([setcap not found])
     fi

